unittests/test_rigid_transformations.py

```diff
@@ -0,0 +1,78 @@
+"""Unit tests for rigid_transformations module."""
```

Comment:
so the canonical syntax for tests is: 

in the same directory, create a file that starts with the module that you want to test and suffix _test.py

this will work with self discovery for the unit test library

Reply:
Cool, thanks.